scripts/convert-docs.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { writeFile } = require('fs/promises');
+const apiDocs = require('../src/api-docs');
+
+(async () => {
```

Comment:
Why are we using async and not the sync version of the `writeFile` api?

Reply:
Just out of habit, I'd say there's no reason to choose one over the other in this specific case.